setup.py

```diff
@@ -72,7 +72,7 @@ def find_version(*file_paths):
     ],
     'all': [],
     'report-timings': [
-        'pandas==0.25.*'
+        'pandas==1.*'
```

Comment:
This dependency was pinned really low, we don't have any tests for `cylc report-timings` but after a quick manual test everything seems to be working fine in `1.1` so might as well get with the times.